DeviceAdapters/FakeCamera/FakeCamera.cpp

```diff
@@ -58,9 +62,13 @@ FakeCamera::FakeCamera() :
 	CreateProperty("Tiff Stack", "0", MM::Integer, false, new CPropertyAction(this, &FakeCamera::OnTiffStack));
 	SetAllowedValues("Tiff Stack", allowedValues);
 
+	//allowedValues.clear();

```

Comment:
clean up pls
Do you have an idea what that did? I cant recreate my reasoning for clearing a vector that will not be used again :D

Reply:
I added this line cause I saw it somewhere else, but did not  need it anyway. its removed now!